src/DroneSafetyApi/Data/HazardDatabase.cs

```diff
@@ -37,11 +37,9 @@ public IEnumerable<Hazard> GetHazardsInRadius(Point location, int radius, DateTi
         private IEnumerable<Hazard> GetShapedHazardsInRadius<T>(Point location, int radius, DateTime time, string ShapeName) where T : Hazard
         {
             var query = client
-                .CreateDocumentQuery<Hazard>(collection.SelfLink, new FeedOptions { EnableScanInQuery = true })
+                .CreateDocumentQuery<T>(collection.SelfLink, new FeedOptions { EnableScanInQuery = true })
                 .Where(c => c.Location.Distance(location) < radius && c.Shape == ShapeName
-                    && ((c.StartTime.CompareTo(time.AddHours(-0.5)) >= 0 && c.StartTime.CompareTo(time.AddHours(0.5)) <= 0)
-                    || (c.EndTime.CompareTo(time.AddHours(-0.5)) >= 0 && c.EndTime.CompareTo(time.AddHours(0.5)) <= 0)
-                    || (c.StartTime.CompareTo(time.AddHours(-0.5)) <= 0 && c.EndTime.CompareTo(time.AddHours(0.5)) >= 0)));
```

Comment:
The hazards are only valid from StartTime to EndTime. If the time requested falls within that range then we want that hazard. Otherwise we could get multiple of the same. For instance, each WiFi hotspot adds data every minute so if it adds (StartTime-EndTime):
```
1: 13:30-13:31
2: 13:31-13:32
3: 13:32-13:33
```
Then we request the hazards at 13:32:30 we only want 3 selected.